pkg/api/agentPoolOnlyApi/v20180331/types.go

```diff
@@ -40,6 +40,7 @@ type Properties struct {
 	AccessProfiles          map[string]AccessProfile `json:"accessProfiles,omitempty"`
 	AddonProfiles           map[string]AddonProfile  `json:"addonProfiles,omitempty"`
 	NodeResourceGroup       string                   `json:"nodeResourceGroup,omitempty"`
+	EnableRBAC              bool                     `json:"enableRBAC"`
```

Comment:
does omitempty matter here?

Reply:
1. should this be on by default?
2. can this be changed? (turning on and off?)

Reply:
Change to *bool, so we have 3 state

nil -> true
true -> true
false -> false

Note, we default behavior is to set it to true.